config.toml

```diff
@@ -25,6 +25,51 @@ home = [ "HTML", "RSS", "JSON"]
   themeVariant = "red"
   disableInlineCopyToClipBoard = "false"
 
+## Use those parameters with {{< param "param_name" >}}
+
+#  deployment = "agnosticd"
+#  external_domain = "<GUID>.${LABID}.opentlc.com"
+#  internal_domain = "<GUID>.internal"
+#  internal_control = "bastion.<GUID>.internal"
+#  internal_host1 = "node1.<GUID>.internal"
+#  internal_host2 = "node2.<GUID>.internal"
+#  internal_isonode = "worker1.emea.<GUID>.internal"
+#  internal_tower1 = "tower1.<GUID>.internal"
+#  internal_tower2 = "tower2.<GUID>.internal"
+#  internal_tower3 = "tower3.<GUID>.internal"
+#  internal_towerdb = "support1.<GUID>.internal"
+#  external_tower = "tower.<GUID>.${LABID}.opentlc.com"
+#  control_prompt = "ec2-user@bastion"
+#  ansible_prompt = "root@bastion"
+#  pre_awx_prompt = "TBD"
+#  awx_prompt = "TBD"
+#  content_git_uri = "structured-content.git"
+
+  deployment = "linklight"
```

Comment:
what's the purpose of variable deployment?

Reply:
I thought, I could also do conditional text `if deployment == linklight elif deployment == ...` but I haven't yet found out how to do it. It might be impossible.